### PR TITLE
CRVS bug fixes

### PR DESCRIPTION
**Pull request by qaziabubakar-vd**
https://github.com/opensrp/opensrp-client-chw/issues/2047
https://github.com/opensrp/opensrp-client-chw/issues/2048

### CODE REVIEW

**Review by allan-on — CHANGES_REQUESTED** [review]
See review comments

File: opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json [hunk 1]
```diff
@@ -164,7 +164,6 @@
         "key": "birth_cert_num",
         "openmrs_entity_parent": "",
         "openmrs_entity": "concept",
-        "openmrs_entity_id": "162052AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (allan-on) [inline opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json:167]: Why was the entity_id removed?

File: opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json [hunk 2]
```diff
@@ -164,7 +164,6 @@
         "key": "birth_cert_num",
         "openmrs_entity_parent": "",
         "openmrs_entity": "concept",
```
Comment (allan-on) [inline opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json:166]: Why was the entity_id removed?

File: opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json (hunk 1 above)
Comment (allan-on) [inline opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json:167]: Why ws this entity_id removed?

File: opensrp-chw/src/crvs/assets/json.form/out_of_area_death_form.json [hunk 3]
```diff
@@ -740,6 +740,75 @@
...
+        "key": "death_certificate_number_volume",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "deathCertificationNumberVolume",
+        "label": "Volume",
+        "hint": "Volume",
+        "type": "edit_text",
+        "v_regex": {
+          "value": "([0-9]{1,4}\\/[0-9]{1,4})",
+          "err": "Number must take the format of ##/####"
```
[the remark this answers is not in the document]
Comment (qaziabubakar-vd) [reply opensrp-chw/src/crvs/assets/json.form/out_of_area_death_form.json]: Relevance is added

File: opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json (hunk 1 above)
Comment (qaziabubakar-vd) [reply opensrp-chw/src/crvs/assets/json.form/birth_certification_changed.json]: Relevance is added

**Pull request merged**